lib/roadie/rails.rb

```diff
@@ -3,6 +3,8 @@ module Rails
   end
 end
 
+require "active_support/concern"
```

Comment:
Any bad side effects for using `ActiveSupport::Concern`?
I am curious (since I use in my gems as well)